app/views/components/update_address_info.scala.html

```diff
@@ -14,12 +14,10 @@
  * limitations under the License.
  *@
 
-@(
-        updateAddressInfo: Boolean
-)(implicit messages: Messages)
+@()(implicit messages: Messages)
 
-@if(!updateAddressInfo) {
-  <h2>@messages("confirmation.updateAddressInfo")</h2>
-  <p>@messages("confirmation.addressChange")</p>
-  <a class="button" id="updateAddressInfoBtn" href="https://www.tax.service.gov.uk/personal-account/personal-details" target="_blank">@messages("confirmation.updateAddressInfoNow")</a>
-}
\ No newline at end of file
+<h2>@messages("confirmation.updateAddressInfo")</h2>
+<p>@messages("confirmation.addressChange")</p>
+<a class="button" id="updateAddressInfoBtn" href="https://www.tax.service.gov.uk/personal-account/personal-details" target="_blank">
```

Comment:
Shouldn't this be moved to config rather than hard coded in the component?

Reply:
Done